site/en/codelabs/openthread-border-router/index.lab.md

```diff
@@ -212,7 +212,7 @@ Done
 > The example output of the `netdata show` command gives
 > 
 > * an OMR prefix `fd76:a5d1:fcb0:1707::/64` for creating an OMR address that can be reached from the Wi-Fi/Ethernet network (you may notice that there is an IPv6 address `fd76:a5d1:fcb0:1707:f3c7:d88c:efd1:24a9` that starts with the OMR prefix);
-> * an external route for the on-link prefix `fd49:7770:7fc5:0::/64` which is created by OTBR when no IPv6 network is already present on the Wi-Fi/Ethernet link.
+> * an external route for the on-link prefix `fd49:7770:7fc5:0::/64` which is created by OTBR when no IPv6 network is already present on the Wi-Fi/Ethernet link. More recent commits will always publish `fc00::/7` in netdata instead of the specific on-link prefix; `fd49:7770:7fc5:0::/64` in this case.
```

Comment:
What about simplifying to something like:
```suggestion
> * an external route - `fc00::/7` if the border router does not have an upstream default route or `::/0` if the border router has an upstream default route.
```
This is the expected behavior for all Thread BRs going forward.

Reply:
sure that sounds good assuming people only use latest commits , i would also have to change the output of the "net data show". I guess its better to put fc00::/7 than just ::/0  there.